conda_lock/conda_lock.py

```diff
@@ -768,7 +768,7 @@ def create_lockfile_from_spec(
     *,
     conda: PathLike,
     spec: LockSpecification,
-    platforms: List[str] = [],
+    platforms: Optional[List[str]] = None,
```

Comment:
Thinking out loud: This is probably safe as-is because this function would only be called one time per CLI invocation. But if that assumption changes, this use of a mutable default value could bite in the future.